internal/daemon/controller/controller.go

```diff
@@ -147,6 +147,7 @@ type Controller struct {
 	PluginRepoFn              common.PluginRepoFactory
 	TargetRepoFn              target.RepositoryFactory
 	WorkerAuthRepoStorageFn   common.WorkerAuthRepoStorageFactory
+	TargetAliasRepoFn         common.TargetAliasRepoFactory
 
 	scheduler *scheduler.Scheduler
```

Comment:
Need to also add this around line 452:
```	
c.TargetAliasRepoFn = func() (*target2.Repository, error){
		return target2.NewRepository(ctx, dbase, dbase, c.kms)
	}
```

Reply:
I'm also running into failures on `TestHandleImplementedPaths` for PATCH and DELETE targets; trying to figure that out, but posting in case you might know why